datumaro/datumaro/plugins/yolo_format/extractor.py

```diff
@@ -99,25 +100,38 @@ def __init__(self, config_path, image_info=None):
 
         self._categories = {
             AnnotationType.label:
-                self._load_categories(self._make_local_path(names_path))
+                self._load_categories(
+                    osp.join(self._path, self.localize_path(names_path)))
         }
 
-    def _make_local_path(self, path):
+    @staticmethod
+    def localize_path(path):
+        path = path.strip()
         default_base = osp.join('data', '')
         if path.startswith(default_base): # default path
             path = path[len(default_base) : ]
-        return osp.join(self._path, path) # relative or absolute path
+        return path
+
+    @classmethod
+    def name_from_path(cls, path):
+        path = cls.localize_path(path)
+        parts = split_path(path)
+        if 1 < len(parts) and not osp.isabs(path):
+            # NOTE: when path is like [data/]<subset_obj>/<image_name>
+            # drop everything but <image name>
+            # <image name> can be <a/b/c/filename.ext>, so no just basename()
+            path = osp.join(*parts[1:])
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'a' in function call](https://app.codacy.com/manual/cvat/cvat/pullRequest?prid=5701448)